cirq/devices/noise_model.py

```diff
@@ -264,3 +264,19 @@ def noisy_moment(
     `cirq.ConstantQubitNoiseModel`.
     """,
 )
+
+
+def validate_all_measurements(moment: 'cirq.Moment') -> bool:
```

Comment:
do we want this to be a public function?

Reply:
totally up for discussion. It is used in both places (cirq.google and cirq.contrib) - we either make it public, duplicate it, or make it private but still call into it? It's not imported into the cirq namespace so it is somewhat "package private".